pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -758,6 +758,9 @@ func TestGetContainerUsageNanoCores(t *testing.T) {
 	var value0 uint64
 	var value1 uint64 = 10000000000
 
+	// Test with a large container of 100+ CPUs
+	var value2 uint64 = 188427786383
```

Comment:
Might be smart to run the assertions (as long as the ptrs are not null) with the values instead of the pointers, as test results like these don't make that much sense:

```
    cri_stats_provider_test.go:901:                                                                                                                                                                                                                                                                                                                        
                Error Trace:    cri_stats_provider_test.go:901                                                                                                                                                                                                                                                                                             
                Error:          Not equal:                                                                                                                                                                                                                                                                                                                 
                                expected: (*uint64)(0xc0001cc1d0)                                                                                                                                                                                                                                                                                          
                                actual  : (*uint64)(0xc0001cc520)    
```

This makes much more sense to me:
```
                                expected: 0x2bdf2bd88f                                                                                                                                                                                                                                                                                                     
                                actual  : 0x2bc5ac5800    
```

Out of scope of this PR tho.